compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1077,17 +1077,8 @@ setRootConfig(func::FuncOp entryPointFn,
       getMatmulVectorSizes(entryPointFn, linalgOp, vectorSize, isQuantized);
 
   DistributionHeuristicConfig distConfig;
-  auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(entryPointFn);
-
-  // Use the default distribution for the matmul loops.
-  int64_t defaultMaxSize = defaultDistTileSize;
-  if (isX86(targetAttr) || isRISCV(targetAttr) ||
-      (isAArch64(targetAttr) && hasAnySVEFeature(targetAttr))) {
-    defaultMaxSize = 128;
-  }
```

Comment:
I don't have context about why this is added for targets with SVE features. @MacDue @banach-space does the change look okay to you?

Reply:
I added `(isAArch64(targetAttr) && hasAnySVEFeature(targetAttr)` because I wanted to align SVE with other targets without touching the defaults for AArch64, i.e. NEON. I don't know why NEON/AArch64 wasn't included here to begin with. 

This change makes sense to me - we should avoid "magic" numbers and unify as much as possible.

Reply:
SG, thanks for confirming! (I am trying to remove magic numbers/logics from the file, and I will keep you in loop.)